src/index.html

```diff
@@ -11,7 +11,137 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="Create HTML page with form. On form submit send form data to https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
```

Comment:
Add indents for attributes if you put them on a newline, make sure to do it for all such tags! Also, remove this text from `action`
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```